doc/topics/packaging/index.rst

```diff
@@ -10,10 +10,78 @@ Relenv onedir packaging
 Starting in 3006, only onedir packaging will be available. The 3006 onedir packages
 are built with the `relenv <https://github.com/saltstack/relative-environment-for-python>`_ tool.
 
+
+Docker Containers
+=================
+We use docker containers to build our packages. If you are building your own packages you can use
+the same containers we build with in the Github piplines. These containers are documented `here <https://github.com/saltstack/salt-ci-containers/tree/main/custom/packaging>`_.
+
+
+How to build onedir only
+========================
+
+Install relenv:
+
+.. code-block:: bash
+
+   # pip install relenv
+
+Fetch toolchain (Only required for linux OSs)
+
+.. code-block:: bash
+
+   # relenv toolchain fetch --arch=<architecture>
+
+Fetch Native Python Build:
+
+.. code-block:: bash
+
+   # relenv fetch --arch=<architecture>  --python=<pythonversion>
+
+Create relenv environment:
+
+.. code-block:: bash
+
+   # relenv create --arch=x86_64 --python=3.10.10 <relenv name>
+
+Add Salt into onedir.
+
+.. code-block:: bash
+
+   # path/to/<relenv-name>/bin/pip install /path/to/salt
+
+
 How to build rpm packages
 =========================
+Install the dependencies:
+
+.. code-block:: bash
+
+   # yum -y install python3 python3-pip openssl git rpmdevtools rpmlint systemd-units libxcrypt-compat git
 
-You only need to run rpmbuild in the Salt repo:
+If you want to build for a specific Salt version, you will need to install tools and changelog dependencies:
```

Comment:
**Suggestion (phrasing, non-blocking):** Instead of "If you want to build..." maybe instead say "To build a specific Salt version..."